release.sh

```diff
@@ -116,9 +116,9 @@ sed -i.bak -E '/^\/dist\/$/d' .gitignore
 rm -f .gitignore.bak
 git add .gitignore dist/
 
-# Edit docs to use CSS from CDN (#1520)
-# grep -l '{@stylesheet: static/katex.min.css}' docs/*.md | xargs sed -i.bak \
-#     's|{@stylesheet: static/katex.min.css}|<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha256-katex.min.css" crossorigin="anonymous"/>|'
+# Edit docs to use CSS from CDN
+grep -l '{@stylesheet: katex.min.css}' docs/*.md | xargs sed -i.bak \
+    's|{@stylesheet: katex.min.css}|<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha384-katex.min.css" crossorigin="anonymous"/>|'
```

Comment:
Does the plugin inject the actual SHA384 for `katex.min.css`?

Reply:
@kevinbarabash The `update-sri.js` script updates the hash.